framework/Source/Operations/LookupFilter.swift

```diff
@@ -1,15 +1,39 @@
+import Foundation
+
 public class LookupFilter: BasicOperation {
     public var intensity:Float = 1.0 { didSet { uniformSettings["intensity"] = intensity } }
     public var lookupImage:PictureInput? { // TODO: Check for retain cycles in all cases here
         didSet {
+            inputTextures.removeValue(forKey: 1)
```

Comment:
Suggestion: We can make the key `1` a constant to avoid a magic number. This constant can also be used in `newTextureAvailable`.

Reply:
Do you mean like this?
```Swift
var secondSourceIndex: UInt = 1

inputTextures.removeValue(forKey: secondSourceIndex)

if fromSourceIndex == secondSourceIndex { }
```

Reply:
Does `1` simply mean the second, but not a magic number?

I guess it might me who's not too familiar with this repo. I was just not able to tell why we look for `1`.

Reply:
If the semantic was simply "the second" then we don't need the constant.

Reply:
No it's not a magic number, it's just the second source.
We will be able to access the texture here:
https://github.com/cardinalblue/GPUImage3/blob/master/framework/Source/Operations/LookupFilter.metal#L12